apps/yggdrasil/lib/yggdrasil/role_permission.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Yggdrasil.RolePermission do
+  use Ecto.Schema
+
+  @primary_key false
+  schema "roles_permissions" do
+    field :resource, :string
+    field :permission, :string
```

Comment:
Cool, I think it also improves the usability. Now if we end up adding extra fields to a `permission` or `resource` things might change, but I suppose you could always look up the record separately from cache or something.